src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1618,7 +1618,7 @@ private static IEnumerable<CommandLineAnalyzerReference> ParseAnalyzers(string a
                 yield break;
             }
 
-            List<string> paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path)).ToList();
+            IEnumerable<string> paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path));
```

Comment:
💭 This one has the downside of making it harder to see the result while debugging. Is it really that big of a win?